google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -806,7 +806,7 @@ Object getValue() {
     }
 
     /**
-     * The HTTP method to be used with the signed URL.
+     * The HTTP method to be used with the signed URL. Defaults to GET.
```

Comment:
Does it make sense to say "Defaults to GET" for a setter method? In what case would this method be called, but the default GET is used?

Reply:
I'm trying to communicate that GET is the default method if this isn't called, not quite sure what wording I should use here? Maybe this?

> The HTTP method to be used with the signed URL. If this is not set, defaults to GET.

Reply:
Sounds good!

Reply:
Done, ready for re-review I think?